src/frontend/src/components/RecordSearch/SearchPanel/index.tsx

```diff
@@ -127,7 +128,7 @@ export default class SearchPanel extends React.Component<Props, State> {
         <section className="cf mt4 mb3 pa4 bg-white shadow br3">
           <form className="mw7 center" onSubmit={this.handleSubmit} noValidate>
             {aliasComponents}
-            <div className="flex">
+            <div className="flex mb4">
```

Comment:
This adds extra margin on the bottom of the search form when an error isn't present.

One thing we could do is remove the bottom padding on the parent `<section>` to make up for it, so `ph4 pt4` instead of `pa4`... 

Actually, that looks like it will give an awkward amount of margin above the error (`mb4`), so I'm thinking a better approach may be adding some top margin to the error component (probably `mv3`) so we don't have to have the spacing on the error containers.

Reply:
I moved the spacing to only appear on the error banner, but it makes the ones on the OECI login to look awkwardly spaced... dang. This refactor might not actually be worth it?